types/relay-runtime/index.d.ts

```diff
@@ -170,9 +170,7 @@ export { readInlineData } from './lib/store/readInlineData';
 
 // Extensions
 export { RelayDefaultHandlerProvider as DefaultHandlerProvider } from './lib/handlers/RelayDefaultHandlerProvider';
-export {
-    missingViewerFieldHandler as DefaultMissingFieldHandlers,
-} from './lib/handlers/RelayDefaultMissingFieldHandlers';
+export { DefaultMissingFieldHandlers } from './lib/handlers/RelayDefaultMissingFieldHandlers';
```

Comment:
3. After which [the below suggested change] this should be:

```suggestion
import DefaultMissingFieldHandlers from './lib/handlers/RelayDefaultMissingFieldHandlers';
export { DefaultMissingFieldHandlers };
```